python/test/test.py

```diff
@@ -0,0 +1,43 @@
+from ctypes import *
+import unittest
+from evaluator5 import evaluate_5cards
```

Comment:
Import `evaluator5` doesn't seem to work unless I move all the source files and the test file to the same folder.

Would you be able to fix this part? I found this [repo](https://github.com/navdeep-G/samplemod/blob/master/tests/context.py) might be helpful.